docs/snapshotters/erofs.md

```diff
@@ -128,6 +128,32 @@ $ # run the container with the provides snapshotter
 $ ctr run -rm -t --snapshotter erofs docker.io/library/busybox:latest hello sh
 ```
 
+## Data Integrity
+
+The EROFS snapshotter provides two methods to consolidate data integrity:
+
+### Data Integrity with Immutable File Attribute
+
+By setting `set_immutable = true`, the EROFS snapshotter marks each layer blob
+with IMMUTABLE_FL. This ensures that dirty data is flushed immediately and the
+EROFS layer blob cannot be deleted, renamed, or modified.
+
+Since it can flush in-memory dirty data, it may greatly impact container launch
```

Comment:
Do you have benchmark data?

Reply:
Hi! See the PR description and I’ve listed the numbers but since it could be varied between erofs-utils and environment I avoid listing in the doc

Reply:
The way this is worded is ambiguous as to *how* performance is impacted.
Consider rewording like "may significantly increase the time it takes to launch a container" and show some example data that we've seen.

Reply:
added.